src/api/developer.ts

```diff
@@ -7,13 +8,19 @@ import {
 export const getDeveloperCollabration = async (developer: string) => {
   return (
     mockSuccessRes(developerCollabrationData) ||
-    (await request(`/actor/${developer}.json`))
+    (await request(`${HYPERTRONS_OSS_XLAB_ENDPOINT}/actor/${developer}.json`))
   );
 };
 
 export const getParticipatedProjects = async (developer: string) => {
   return (
     mockSuccessRes(participatedProjectsData) ||
-    (await request(`/actor/${developer}_top.json`))
+    (await request(
+      `${HYPERTRONS_OSS_XLAB_ENDPOINT}/actor/${developer}_top.json`
```

Comment:
What is developer_top.json? I don't think the file is online right now

Reply:
developer_top.json is the data source of our graph charts and they are stored in hypertrons-oss.xlab.info. There are two data endpoints now.

Reply:
Yes, but I think you use the same endpoints here or the `HYPERTRONS_OSS_XLAB_ENDPOINT` is different ?

Reply:
My fault, misunderstand here.